addins/Cake.Xamarin.Build.yml

```diff
@@ -0,0 +1,15 @@
+Name: Cake.Xamarin.Build
+NuGet: Cake.Xamarin.Build
+Assemblies:
+- "/**/Cake.Xamarin.Build.dll"
+Repository: https://github.com/Redth/Cake.Xamarin.Build
+Author: Redth
+Description: "NOT RECOMMENDED FOR PUBLIC USE: Cake Build addon to provide Aliases for Xamarin product build scripts.
```

Comment:
I think this should also be filtered, as it is obviously not the intend of the author to have it used

Reply:
Done. Cake.Xamarin.Build added to blacklist